<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEN_LancerCourse</class>
 <widget class="QDialog" name="FEN_LancerCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEN_LancerCourse</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_LaunchRace">
      <property name="text">
       <string>Lancer la course !</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_hares">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Lievres</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spBx_turtles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>50</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>87</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_turtles">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Tortues</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spBx_hares"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
